### PR TITLE
Add support for ArseLinux

### PR DESCRIPTION
**Pull request by LiamAEdwards**
## Description

Only fill in the fields below if relevant.

### CODE REVIEW

**Comment by hykilpikonna** [comment]
lol ok

File: neofetch [hunk 1]
```diff
@@ -5768,6 +5768,28 @@ ${c1}         -o          o-
 EOF
         ;;
 
+                "ArseLinux"*)
```
Comment (hykilpikonna) [inline neofetch:5771]: Please try to be consistent with the indents though

```suggestion
        "ArseLinux"*)
```

**Comment by hykilpikonna** [comment]
Surely does look like an arse

image

**Comment by hykilpikonna** [comment]
Thank you for your contribution! 

This PR is [merged into hyfetch](https://github.com/hykilpikonna/hyfetch/commit/de50bfdb) since this repo (dylanaraps/neofetch) seems no longer maintained.

[HyFetch](https://github.com/hykilpikonna/hyfetch) is a fork of neofetch with LGBTQ pride flags, but the repo also maintains an updated version of the original neofetch, addressing many pull requests that are not merged in the original repo.

Read the ["Running Updated Original Neofetch" section](https://github.com/hykilpikonna/hyfetch#running-updated-original-neofetch) for more info!

**Comment by LiamAEdwards** [comment]
Peachy arse indeed! Thank you :)

**Pull request closed without merging**